src/rdo.rs

```diff
@@ -637,6 +637,14 @@ impl fmt::Debug for DistortionScale {
   }
 }
 
+impl From<f64> for DistortionScale {
+  #[inline]
+  fn from(scale: f64) -> Self {
+    let den = 1 << (Self::SHIFT + 1);
+    Self::new((scale * den as f64) as u64, den)
```

Comment:
Is this going to result in any rounding issues that we care about?

Reply:
This is equivalent to the previous rounding behavior. However, there is an issue with overflow for extremely large values.

Reply:
What is the likelihood of any user encountering the overflow?

Reply:
I'll fix it since u64::saturating_add is const